python-main.js

```diff
@@ -1331,16 +1329,21 @@ function web_editor(config) {
         $("#command-share").click(function () {
             doShare();
         });
-        $("#command-connect").click(function () {
-            if ($("#command-connect").length) {
-                doConnect();
-            } else {
-                doDisconnect();
-            }
-        });
-        $("#command-serial").click(function () {
-            doSerial();
-        });
+        if (navigator.usb) {
+            $("#command-serial").click(function () {
+                doSerial();
+            });  
+            $("#command-connect").click(function () {
+                if ($("#command-connect").length) {
+                    doConnect();
+                } else {
+                    doDisconnect();
+                }
+            }); 
+        }else{
+            document.getElementById('command-serial').setAttribute('href',"help.html#WebUSB");
```

Comment:
We're already using jQuery elsewhere so it may be more in-style to use jQuery selectors

`$("#id").attr("href","url_goes_here");`